eng/pipelines/azure-pipelines.yml

```diff
@@ -88,4 +87,8 @@ stages:
             displayName: Unix Build / Publish
 
 - ${{ if eq(variables._RunAsInternal, True) }}:
-  - template: eng\common\templates\post-build\post-build.yml
\ No newline at end of file
+  - template: eng\common\templates\post-build\post-build.yml

```

Comment:
What feed is this pipeline going to publish to? 

I think we shouldn't be publishing to dotnet-core, should we?

Reply:
This publishes to the channel's corresponding feed, dotnet5-transport.

Reply:
Cool. Good to know it doesn’t publish to dotnet-core.